schema.txt

```diff
@@ -0,0 +1,27 @@
+Teams - represents a team
+-> Id
+-> [UserIds]
+-> Name
+-> Header
+-> Description
+-> ProjectEndDate - currently working on a project
+-> CanStartAtMin - minimum notice period of the team member
+-> CanStartAtMax - minimum notice period of the team member
+-> [TagIds]
+
+-Accounts - represents a teampot account
+-> Id
+-> [TeamIds]
+-> First Name
+-> Last Name
+-> Role
+-> NoticePeriod
+-> Description
+-> [{SocialAccounts}] - social media account (LinkedIn, githib, ...)
```

Comment:
Is the Social Media account also an Entity?

Reply:
Probably add the CreationDate UpdateDate props, just for auditing for all

Reply:
Yep, we should add dates for tracking purposes:
-CreationDate
-CreationUser
-EditDate
-EditUser

for all entities?

Reply:
We put social media account for now as a property of UserAccount, since its doesn't grow infinitely and each user will have a couple of Social Accounts. Would you separate it and point to UserAccount?

Reply:
my 2 cents is that for now its ok as it is, and array of accounts, although not sure at all how we can merge `LinkedIn` or `github` data as it seems quite diferent